README.md

```diff
@@ -0,0 +1,5 @@
+# Plexamp
+
+Flatpak wrapper around [Plexamp](https://plexamp.com).
+
+Note that using Plexamp **requires** a [Plex Pass](https://www.plex.tv/plex-pass/).
```

Comment:
End users don't really see this, we should put the information in the appdata description tag.